processes/vuln_db.md

```diff
@@ -0,0 +1,99 @@
+  "overview": "attack overview\n\n"
+}
+```
+
+### Node.js Ecosystem vulnerabiltiy report structure
+
+```json
+{
+  "id": XXX,
+  "title": "Denial of Service",
+  "overview": "attack overview",
+  "created_at": "2018-02-22",
+  "updated_at": "2018-02-22",
+  "publish_date": "2018-02-22",
+  "author": "John Smith (https://www.github.com/john-smith)",
+  "module_name": "some-module-name",
+  "cves": [
+    "CVE-YYYY-XXXX"
+  ],
+  "vulnerable_versions": "<1.0.0",
+  "patched_versions": ">=1.0.0",
+  "recommendation": "Update to version 1.0.0 or later",
+  "references": [
+    "https://hackerone.com/reports/XXXX",
+  ],
+  "cvss_vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
+  "cvss_score": 6.5,
+  "coordinating_vendor": null
+}
+```
+
+### Validating reports
+
+The Security WG employs a static code analysis tool that runs in CI for all pull requests to validate the correct structure when new vulnerabilities are introduced.
+
+You may make use of the same tool which is available [here](https://github.com/nodejs/security-wg/blob/master/tools/vuln_valid/index.js) either in your own tooling setup or from this repository by running `npm test`.
+
+
+### Database structure changes
+
+To support a maintainable and enhanced vulnerabilities database, the Security WG may make changes to the structure of existing reports in order to make proper accommodations to evolving needs of the community.
+
+As we set out to re-structure the vulnerability database we will do our best to communicate the change broadly as possible and provide a reasonable time to get involved in the discussion.
+
+We will announce an upcoming change in the following ways:
+- A new issue in the repository (see example reference: https://github.com/nodejs/security-wg/issues/345)
+- We will contact the list of known database users (see section below) and alert them of the change, inviting them to participate in the discussion before changes are merged.
+- By social media, using twitter, to further broadcast the change through some of the Security WG's twitter accounts.
+
+Time window for introducing changes:
+- Changes that affect existing database fields will provide a 30 days time-frame through announcement for any party to chime in and weigh on the changes proposed
+- Changes that extend the database fields with newly proposed fields will be added as the proposed pull request is merged
+
+Note: we recommended to subscribe to the repository's issue tracker in order to receive updates in a timely manner for both new issues that may influence database changes, as well as new vulnerabilities pushed to the database or changes made to existing. Also, all pull request that add a new vulnerability are labled with `vulnerabilities` which makes them easily filtered and searchable.
+
+
+## Database Users
+
+The following is a list of users or organizations which opt-in to be listed as users of the vulnerability database:
+
+- [ ] Customer 1 (contact: customer@example.com)
```

Comment:
I'll be interested to see if people are willing to list emails in a public doc.  We may need these to be github ids, but lest see what people think.

Reply:
right. I was thinking some of these will actually be organizations like snyk, etc so you'd probably have an official e-email there.